scripts/pgdata13.sh

```diff
@@ -0,0 +1,50 @@
+#!/hab/pkgs/core/bash/5.1/20210826055113/bin/bash
```

Comment:
this too, please confirm if this is pinning is needed?

Reply:
changed it to /bin/bash